dune-project

```diff
@@ -2,3 +2,4 @@
 (name index)
 (using fmt 1.1)
 (implicit_transitive_deps false)
+(allow_approximate_merlin)
```

Comment:
What's this for?

Reply:
I had [this warning](https://discuss.ocaml.org/t/new-dune-warning-merlin-generated-is-inaccurate-pps-specification-isnt-identical-in-all-stanzas/3760) for which I don't know if there is another fix. Anyway, it does not matter anymore, since I remove the second executable (`accuracy`) from the `bench` directory, which was causing the problem.